docs/userguide.md

```diff
@@ -712,3 +713,15 @@ A and B are offline:
 ```
 Replica layout: C=20
 ```
+
+## Controller-Manager Leader Election
+
+The federation controller manager is always deployed with leader election feature
```

Comment:
We deploy with leader election by default, but replica count is [defaulted to 1](https://github.com/kubernetes-sigs/federation-v2/blob/master/charts/federation-v2/values.yaml#L9).  Should this be updated to 2 by default?  Otherwise I'm not sure this doc is very accurate since we might as well not be enabling by default.

Reply:
yes, i think we should update default replica count to 2, as that's how one is supposed to deploy in production. I would add a commit to update default replica count.